R/install.R

```diff
@@ -378,3 +401,4 @@ get_install_libs_url <- function(version = "1.9.1", type = install_type(version
   list(libtorch = libtorch, liblantern = liblantern)
 }
 
+
```

Comment:
I'd like to remove all whitespaces and empty lines, after merging the pending PRs.

Reply:
That would be great!

Reply:
I'm on it. It will be a relatively big change for code cleanup, styling, and formatting.